### ISSUE TITLE
Show big visual indicator if Singularity loses connectivity with Mesos

### ISSUE THREAD

**Issue by tpetr** [HubSpot/Singularity#776]
If Singularity has issues (re)registering with Mesos, we'll only know about it via the `mesos running:` part of log lines like this:

```
TRACE [2015-11-23 23:50:32,160] com.hubspot.singularity.scheduler.SingularityLeaderOnlyPoller: Skipping SingularitySchedulerPoller (period: 00:05.000) (leadership: true, mesos running: false, enabled: true)
```

We need to:
- Expose this value in the state endpoint
- Make an obnoxious visual indicator in the UI so that users know something is up
- Link to documention on how to fix

/cc @scalp42

**Comment by scalp42** [comment]
thanks again for the help @tpetr !

**Comment by ssalinas** [comment]
Implemented this in #965 , feel free to comment if you have any suggestions for the actual warning message itself
